src/greyVest/ExpandableTable.js

```diff
@@ -50,61 +54,89 @@ let TableBodyState = () => {
 }
 
 let TableBody = inject(TableBodyState)(
-  observer(({ data, columns, recordKey, expanded, onClick }) => (
-    <tbody>
-      {_.map(
-        x => (
-          <React.Fragment key={x[recordKey]}>
-            <tr
-              {...x.rowAttrs}
-              key={x[recordKey]}
-              className={
-                _.getOr('', 'rowAttrs.className', x) +
-                (expanded.has(x[recordKey]) ? 'expanded' : '')
-              }
-            >
-              {F.mapIndexed(
-                ({ field, display = x => x, details = {} }, i) => (
-                  <td
-                    key={`${field}${i}`}
-                    {...(!_.isEmpty(details) && {
-                      style: {
-                        cursor: !_.isEmpty(details) ? 'pointer' : 'auto',
-                      },
-                      onClick: () =>
-                        onClick(field, recordKey, x, i, details, expanded),
-                    })}
-                  >
-                    {_.getOr(
-                      display,
-                      `${
-                        _.isEqual(
-                          _.get('indexedField', expanded.get(x[recordKey])),
-                          `${field}${i}`
-                        )
-                          ? 'collapse'
-                          : 'expand'
-                      }.display`,
-                      details
-                    )(_.get(field, x), x)}
-                  </td>
-                ),
-                columns
-              )}
-            </tr>
-            {/* See if there is a details component to render for the column value when row expanded */}
-            {expanded.has(x[recordKey]) && (
-              <ExpandedSection
-                expandedRow={expanded.get(x[recordKey])}
-                columnCount={columns.length}
-              />
-            )}
-          </React.Fragment>
-        ),
-        data
-      )}
-    </tbody>
-  ))
+  observer(
+    ({
+      data,
+      columns,
+      recordKey,
+      expanded,
+      onClick,
+      limitedResults,
+      pageSize,
+    }) => {
+      let rows = data
+      if (limitedResults && rows.length > 0) {
+        let blankRows = [...Array(pageSize - rows.length)].map((_, i) => ({
+          ...rows[i % rows.length],
+          // unique deterministic IDs
+          [recordKey]: `${rows[i % rows.length][recordKey]}${i}`,
+          isBlank: true,
+        }))
+        rows = [...rows, ...blankRows]
```

Comment:
let's factor out a `fillWithBlanks` method since this is the same as ResultTable (result table's `recordKey` is just `_id`)